CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Changed
 
 - Update architect-orb to 0.10.0
+- Sync with upstream
```

Comment:
Please add more info about version updated, operator and subcharts/components, link to upstream changelog if available.